### PR TITLE
Rectified Issue #298

### PR DESCRIPTION
**Pull request by itsnitish22**
### Types of changes
Rectified the issue of background colour inconsistency on the tap of any item in the balloon of action item on the Top Nav Bar.
Issue #298

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -20,7 +20,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:$versions.gradleBuildTool"
+        classpath 'com.android.tools.build:gradle:7.0.4'
```
Comment (skydoves) [inline build.gradle:23]: Seems not related to this issue.

```suggestion
        classpath "com.android.tools.build:gradle:$versions.gradleBuildTool"
```

File: balloon/build.gradle [hunk 2]
```diff
@@ -24,8 +24,6 @@ android {
     defaultConfig {
         minSdkVersion versions.minSdk
         targetSdkVersion versions.compileSdk
-        versionCode versions.versionCode
-        versionName versions.versionName
```
Comment (skydoves) [inline balloon/build.gradle:28]: Seems not related to this issue.

**Review by skydoves — CHANGES_REQUESTED** [review]
Hey @itsnitish22,
Thanks for making this RP! I left some comments for some changes which are not related to this issue.

**Comment by itsnitish22** [comment]
Seems like it was done by Android Studio. 
I have done the required changes now.

File: build.gradle [hunk 3]
```diff
@@ -20,7 +20,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:$versions.gradleBuildTool"
+      classpath "com.android.tools.build:gradle:$versions.gradleBuildTool"
```
Comment (skydoves) [inline build.gradle:23]: One more tweak. 😉

**Review by skydoves — CHANGES_REQUESTED** [review]
Great job! Here's one more tweak.

File: build.gradle (hunk 3 above)
Comment (itsnitish22) [reply build.gradle:23]: Oops. Will rectify this 😅

**Comment by itsnitish22** [comment]
Hope it works now 😆

**Review by skydoves — APPROVED** [review]
👍
